apps/minifront/src/components/dashboard/assets-table/helpers.ts

```diff
@@ -0,0 +1,15 @@
+import { assetPatterns } from '@penumbra-zone/types/assets';
+import { getDisplay } from '@penumbra-zone/getters/metadata';
+import { BalancesResponse } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1/view_pb';
+import { getMetadata } from '@penumbra-zone/getters/value-view';
+
+const hiddenAssetPatterns = [assetPatterns.auctionNft];
+
+const isUnknown = (balancesResponse: BalancesResponse) =>
+  balancesResponse.balanceView?.valueView.case === 'unknownAssetId';
+
+export const isVisible = (balance: BalancesResponse) =>
```

Comment:
instead of `isVisible` can we do something like `shouldDisplay`? (just so there isn't an confusion this is a private data thing)